### ISSUE TITLE
Detected cheater in pokeadvisor

### ISSUE THREAD

**Issue by ariesfung11** [finndev/PokeBuddy#297]
I login to pokeadvisor and check the account status. 

In the ranking column, it showed "detected cheater", i am worring it will be banned.

Is the website getting data from pokemon go server, or just calculated by themselves?

**Comment by jayhao2000** [comment]
from Pokemon Go server, mate

**Comment by ariesfung11** [comment]
So i should del the account then

**Comment by willsims14** [comment]
How long did it take for you to get that message to appear? I haven't had my rank load yet, all it says is "Rank updating..."

**Comment by dodolock** [comment]
u really think niantic will give their data to a third party called pokeadvisor? use ur brain to think before u talk plz . if u have a brain

**Comment by bcmmbc122333** [comment]
can you share a screenshot please?

**Comment by dodolock** [comment]
and if u are promoting this shitty website pokeadvisor, plz just go hell and meet ur family there

**Comment by Sweg7** [comment]
Hope this helps:
It's a third party site and have no affiliation with Niantic or Nintendo. They probably detect it by experience rate deeming everything above x exp/hour too high and labels it as a cheater. With that said, rest assured Niantic has even better data at their disposal to see who has and hasn't been cheating. Don't bot on an account that means anything to you until we know what action they take. ~Fadelol

http://www.ownedcore.com/forums/pokemon-go/pokemon-go-hacks-cheats/567793-pokeadvisor-labeling-cheater.html

**Comment by bcmmbc122333** [comment]
After a little research I have come to 2 conclusions, 1. is what sweg7 said "They probably detect it by experience rate deeming everything above x exp/hour too high and labels it as a cheater."  I have also heard that it might be off of pokestops/distance traveled. and 2. They are archiving botters for a mass ban. I think it is number 1 personally because some people have been getting permanently banned already which makes me believe that they are not archiving, also because niantic never did a mass ban on ingress.  Either way if you are using pokebuddy without teleporting where the guy walks to each stop you should be ok from getting a botting ban, but if you gps spoofed a long distance who knows, good luck and I hope you dont get banned! :)

**Comment by Sweg7** [comment]
@bcmmbc122333 didnt ingress have a mass ban?

**Comment by bcmmbc122333** [comment]
@Sweg7 Ingress never had a mass ban but when they started perm banning a ton of people got hit right off the bat, and yes they are waiting to fully launch the game and fix bugs before they roll out with perm bans as a common punishment, but a few people have already been perm banned so far.

**Comment by dodolock** [comment]
@bcmmbc122333 no ,i tped a lot since I lvled up to lvl28 manually. And no one got perm banned so far.Teleport to somewhere will not get us banned i think. Because how Niantic distinguish its a gps spoofing or I have my friends logged in with my account in a different place where is far away from my place

**Comment by dodolock** [comment]
@bcmmbc122333  I believe thats the reason why there is no massive perm bans in Ingress. Also it will not happen in PG. However, i think Niantic can find peoplee using bot farming or not. Otherwise, all bots should still work at the time we discussing  here

**Comment by bcmmbc122333** [comment]
@dodolock i have seen numerous people posting perm bans all over Reddit, I guess its all people faking bans for attention.

**Comment by ariesfung11** [comment]
I gain around 70k exp per hour, maybe it caused me to be cheater. But i will still play that account until Niantic take any action. Thanks

**Comment by dodolock** [comment]
@bcmmbc122333 ye, I believe so. And as an game company, their goal is having as many players as possible and make them play as longer as possible. So BANNING account is not their optimal choose. They ban accounts only if these accounts used some tools which affected their goal dramatically

**Comment by Sweg7** [comment]
@dodolock You can get a soft ban for teleporting to much, so we know they can perma ban someone whenever they want to.

**Comment by dodolock** [comment]
@Sweg7 soft ban is a built in game stratege. It requests no info to interact with serve. It is like I put a rule in the game, if u break the rule, the game program itself will stop you playing for a while. That's it

**Comment by dodolock** [comment]
@Sweg7  and not tped to much. If u tped a long distance one time u still got softban ,cuz this "long distance" breaks the built in rules

**Comment by Sweg7** [comment]
Okay, I see that now. But I'm sure they will start perma-banning players soon because it gives players an unfair advantaged. Like without going outside and looking for the pokemon yourself (which the game wasn't designed for) seems pointless to many. This also makes it impossible for the players who played the game legit to beat players that have botted in a gym. One more thing to play into this that once players start getting all 151 pokemon and beating a lot of gyms back and forth the game will become repetitive to them and will become boring and they will stop playing due to no real competition.

**Comment by curtiola** [comment]
6v1, any botter will still get slaughtered in a gym lol though i understand your point

**Comment by dodolock** [comment]
@Sweg7  the problem is they have never listen to players community, and refused to fix the tracking system. Even you know u got a Mew nearby, you have no idea where to catch it. That is the real problem that Niantic should look after right now .  I do understand it is unfair for people not using a bot to lvl up, but to lvl up to 40 manually seems like an impossible mission. They should learn from Bliz, Listen to people or players. and get advises from them. What they done so far is No COMMUNICATION LIKE 0

**Comment by Sweg7** [comment]
@dodolock Agreed. They should listen to the community more but you have to also see that they are having many major troubles in the games like keeping servers from going offline. Once they fix the major issue they can then listen to the community. It seems to me that they weren't expecting this much influx of players when they launched the game.

**Comment by sqrcbc** [comment]
i dont really get the whole QQ about getting possibly banned due to botting and therefore acting against the TOS...
if you bot you SHOULD be aware that you violate the rules, period.
It's pretty much the same in any (nearly) competetive game, whose game mechanics are bottable.
I agree that botters are bannable, yet i am botting myself.
If you get caught, then you deserve it.

the whole botters got an advantage argumentation is pretty one sided btw.
i personally bot to catch up with these guys, who are playing the game all day long.
evry medal got two sides ;)

about the detected cheater... 
this guy is lvl 34 and joined no team...?
fishy by that fact already, imo

**Comment by Sweg7** [comment]
@sqrcbc good point
